ffmpeg_normalize/_ffmpeg_normalize.py

```diff
@@ -163,38 +174,16 @@ def __init__(
         self.debug = debug
         self.progress = progress
 
-        self.stats = []
-
         if (
-            self.output_format
-            and (self.audio_codec is None or "pcm" in self.audio_codec)
-            and self.output_format in PCM_INCOMPATIBLE_FORMATS
-        ):
+            self.audio_codec is None or "pcm" in self.audio_codec
+        ) and self.output_format in PCM_INCOMPATIBLE_FORMATS:
             raise FFmpegNormalizeError(
                 f"Output format {self.output_format} does not support PCM audio. "
-                + "Please choose a suitable audio codec with the -c:a option."
-            )
-
-        if normalization_type not in NORMALIZATION_TYPES:
-            raise FFmpegNormalizeError(
-                f"Normalization type must be one of {NORMALIZATION_TYPES}"
+                "Please choose a suitable audio codec with the -c:a option."
             )
 
-        if self.target_level and not isinstance(self.target_level, Number):
-            raise FFmpegNormalizeError("target_level must be a number")
-
-        if self.loudness_range_target and not isinstance(
-            self.loudness_range_target, Number
-        ):
-            raise FFmpegNormalizeError("loudness_range_target must be a number")
-
-        if self.true_peak and not isinstance(self.true_peak, Number):
-            raise FFmpegNormalizeError("true_peak must be a number")
-
-        if float(target_level) > 0:
-            raise FFmpegNormalizeError("Target level must be below 0")
-
-        self.media_files = []
```

Comment:
Why were these parts removed completely?

Reply:
`check_range` does this now, so these checks are now redundant.  Also, `Number` includes complex numbers and rationals, instead of only `float`s and `int`s